### PR TITLE
clarify that his plugin dsl must be within the declarative pipeline c…

### PR DESCRIPTION
**Pull request by gabemontero**
…losure

@openshift/sig-developer-experience @jorgemoralespou fyi / ptal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,19 @@ DSL Plugin is a Jenkins plugin which aims to provide a readable, concise, compre
...
+
+```groovy
+pipeline {
+
+...
+
+}
+```
+
+directive and closure must contain any use of this plugin's DSL.
```
Comment (bparees) [inline README.md]: it's not needed for "any" use. it's needed when you're using declarative semantics in your pipeline.

Comment (gabemontero) [reply README.md]: good catch

**Comment by gabemontero** [comment]
update pushed

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
[merge]

**Pull request merged**